e2e/cypress/integration/channel/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,368 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import * as MESSAGES from '../../fixtures/messages';
+import users from '../../fixtures/users.json';
+import {generateRandomNumber} from '../../utils';
+
+let channelId = '';
+let newChannel = {};
+
+describe('React to last message shortcut', () => {
+    before(() => {
+        cy.apiLogin(users['user-1'].username);
+        cy.visit('/');
+
+        // # Get the current channels Id for later use such as posting message with other user
+        cy.getCurrentChannelId().then((id) => {
+            channelId = id;
+        });
+
+        // # Create a new channel for later use such as when channel is empty test
+        cy.getCurrentTeamId().then((teamId) => {
+            const newChannelName = `emptychannel-${generateRandomNumber(1, 100)}`;
+            cy.apiCreateChannel(teamId, newChannelName, newChannelName).then((response) => {
+                newChannel = Object.assign({}, response.body);
+            });
+        });
+    });
+
+    it('Should open the emoji picker in the channel view when the focus is not on the center text box', () => {
+        // # Making sure there is at least a message without reaction
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Click anywhere to take focus away from center textbox
+        cy.get('#lhsList').within(() => {
+            cy.findByText('Town Square').click();
+        });
+        cy.wait(TIMEOUTS.TINY);
+
+        // # Emulate react to last message shortcut
+        cy.get('body').type('{ctrl}{shift}\\');
```

Comment:
Please use `cmdOrCtrlShortcut` command which is platform aware in using cmd or ctrl.
```suggestion
        cy.get('body').cmdOrCtrlShortcut('{shift}\\');
```